spec/support/factory_bot.rb

```diff
@@ -0,0 +1,4 @@
+RSpec.configure do |config|
```

Comment:
is this file required?

Reply:
[https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rspec](url)

Reply:
what if you add this in `rails_helper.rb`?